javascript/builtins/Intl.json

```diff
@@ -0,0 +1,1144 @@
+              "support": {
+                "webview_android": {
+                  "version_added": false
+                },
+                "chrome": {
+                  "version_added": "24"
+                },
+                "chrome_android": {
+                  "version_added": "26"
+                },
+                "edge": {
+                  "version_added": true
+                },
+                "edge_mobile": {
+                  "version_added": true
+                },
+                "firefox": {
+                  "version_added": "29"
+                },
+                "firefox_android": {
+                  "version_added": "56"
+                },
+                "ie": {
+                  "version_added": "11"
+                },
+                "ie_mobile": {
+                  "version_added": false
+                },
+                "nodejs": {
+                  "version_added": null
+                },
+                "opera": {
+                  "version_added": "15"
+                },
+                "opera_android": {
+                  "version_added": null
+                },
+                "safari": {
+                  "version_added": "10"
+                },
+                "safari_ios": {
+                  "version_added": "10"
+                }
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          }
+        },
+        "NumberFormat": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat",
+            "support": {
+              "webview_android": {
+                "version_added": false
+              },
+              "chrome": {
+                "version_added": "24"
+              },
+              "chrome_android": {
+                "version_added": "26"
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": "29"
+              },
+              "firefox_android": {
+                "version_added": "56"
+              },
+              "ie": {
+                "version_added": "11"
+              },
+              "ie_mobile": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": null
+              },
+              "opera": {
+                "version_added": "15"
+              },
+              "opera_android": {
+                "version_added": null
+              },
+              "safari": {
+                "version_added": "10"
+              },
+              "safari_ios": {
+                "version_added": "10"
```

Comment:
10.2 according to the page

Reply:
I believe that 10.2 is just the current dot release of 10. We've put 10 anywhere else, too.